test/integration_test_helper.rb

```diff
@@ -96,7 +96,7 @@ def select2(value, attrs)
 
   def wait_for_ajax
     Timeout.timeout(Capybara.default_max_wait_time) do
-      loop until page.evaluate_script('jQuery.active').zero? && page.evaluate_script('window.axiosActive').zero?
+      sleep 0.15 while (page.evaluate_script('jQuery.active')&.zero? && page.evaluate_script('window.axiosActive')&.zero?)
```

Comment:
Looking at this again, I think I misled you. the logic is actually now different. We want to stop when both of those values are either nil or zero, which means that the page isn't waiting for an ajax response. So i think the actual correct logic would be:
```ruby
sleep 0.15 until ([page.evaluate_script('jQuery.active'), page.evaluate_script('window.axiosActive')] - [0, nil]).empty?

Reply:
Oh thanks, I had no idea. Rebased.